INSTALL.md

```diff
@@ -41,43 +27,35 @@ $> docker-compose build
 $> docker-compose up
 ```
 
-If you want to run docker-compose with the production settings
+**Notice**: if you see the error "Can't connect to database" in `souschef_web_1`, try Ctrl+C and re-run `docker-compose up`.
+
+(Optional) Running docker-compose with production settings:
 
 ```
 $> docker-compose -f docker-compose.yml -f docker-compose.prod.yml up
 ```
 
-Docker must be up and running at this point.
-
 ## Django initialization
 
-You need the Docker **container ID** to run the next steps. You can get the container_id by running ```docker ps```.
-Ex: *44fcfefb015e*
+In your console:
 
 ```
-docker exec -it [container_id] bash
-
-# Move to the application root dir
-cd src
-
-# Run development server
-python manage.py runserver 0.0.0.0:8080
+$> docker-compose exec web bash
+```
 
-# Access to the development server
-http://0.0.0.0:8080
+Then you should be inside a container as you can see, e.g., `root@d157a3f57426:/code#`. Then run:
 
-# Access to Nginx server
-http://127.0.0.1
+```
+$> cd src
 
 # Run existing migrations
-python manage.py migrate
+$> python3 manage.py migrate
 
 # Create a user with administrator privileges
-python manage.py createsuperuser
+$> python3 manage.py createsuperuser
 
 # Optional: Load the initial data set
-python3 manage.py loaddata sample_data
-
+$> python3 manage.py loaddata sample_data
```

Comment:
@lingxiaoyang don't need to specify python3 because python-3.5 is the default one inside the container:

> root@6013baddd117:/code# python
> Python 3.5.3 (default, Jan 24 2017, 17:43:13) 
> [GCC 4.9.2] on linux
> Type "help", "copyright", "credits" or "license" for more information.
> >>> 

But is OK for me, it doesn't hurt to be more precise.